server_admin/topics/sessions/administering.adoc

```diff
@@ -6,10 +6,10 @@ If you go to the `Sessions` left menu item you can see a top level view of the n
 .Sessions
 image:{project_images}/sessions.png[]
 
-A list of clients is given and how many active sessions there currently are for that client. You can also logout all
+A list of clients is given and how many active sessions there currently are for that client. You can also log out all
 users in the realm by clicking the `Logout all` button on the right side of this list.
 
-==== Logout All Limitations
+==== Limitations of Logout All
```

Comment:
Perhaps "Limitations of `Logout All`" or "Limitations of the `Logout All` operation" would be better?